syft/__init__.py

```diff
@@ -39,22 +39,19 @@
 # Import Hook
 from syft.frameworks.torch.hook.hook import TorchHook
 
-# Import grids
-from syft.grid.private_grid import PrivateGridNetwork
-from syft.grid.public_grid import PublicGridNetwork
-
-# Import sandbox
-from syft.sandbox import create_sandbox, make_hook
-
-# Import federate learning objects
-from syft.frameworks.torch.fl import FederatedDataset, FederatedDataLoader, BaseDataset
-from syft.federated.train_config import TrainConfig
+# communication interfaces
+import syft.interfaces
```

Comment:
These should probably live in the `serde` package with the other serialization code.

Reply:
I could to that, but it will generate the horror named cyclic dependency, I could hack it with a try and catch, I saw it in a few places around the codebase. (or we want to get rid of such hackst?).